src/transformers/models/deit/modeling_deit.py

```diff
@@ -146,7 +146,7 @@ def transpose_for_scores(self, x):
         x = x.view(*new_x_shape)
         return x.permute(0, 2, 1, 3)
 
-    def forward(self, hidden_states, head_mask=None, output_attentions=False):
+    def forward(self, hidden_states, head_mask=None, output_attentions=False, attention_mask=None):
```

Comment:
```suggestion
    def forward(self, hidden_states, attention_mask=None, head_mask=None, output_attentions=False):
```
For consistency with other models ;)